lib/ex_wire/config.ex

```diff
@@ -4,12 +4,22 @@ defmodule ExWire.Config do
   """
 
   @port Application.get_env(:ex_wire, :port, 30304)
-  @private_key ExthCrypto.ECIES.ECDH.new_ecdh_keypair() |> Tuple.to_list() |> List.last # Application.get_env(:ex_wire, :private_key)
+  @private_key ( case Application.get_env(:ex_wire, :private_key) do
+    key when is_binary(key) -> key
+    :random -> ExthCrypto.ECIES.ECDH.new_ecdh_keypair() |> Tuple.to_list() |> List.last
```

Comment:
Makes me want `ExthCrypto.randomKeyPair()` :)

Reply:
Yeah, I'll add this once we start to handle key management realistically.